sw/airborne/boards/ardrone/board.c

```diff
@@ -31,32 +31,39 @@
 #include <unistd.h>
 #include "mcu.h"
 
+#include "std.h"
+#include "modules/computer_vision/lib/v4l/v4l2.h"
```

Comment:
these includes are not needed, keep the previous #include "peripherals/video_device.h" instead